webservices/tasks/__init__.py

```diff
@@ -10,21 +11,37 @@
 schedule = {}
 if env.app.get('space_name', 'unknown-space').lower() != 'feature':
     schedule = {
+        # Refresh all Materialized views at 9am UTC daily
         'refresh': {
             'task': 'webservices.tasks.refresh.refresh',
-            'schedule': crontab(minute=0, hour=9),
+            'schedule': crontab(minute=0, hour=9), 
+        },
+        
+        # If any changes in AOs during the day all AOs get reloaded at 1am UTC
+        'reload_all_aos': {
+            'task': 'webservices.tasks.legal_docs.reload_all_aos',
+            'schedule': crontab(minute=0, hour=1, day_of_week='mon,tue,wed,thu,fri,sat'),
+        },
+        
+        # All AOs get reloaded every Sunday at 1am UTC whether any changes to AOs or not
+        'reload_all_aos_wkly': {
+            'task': 'webservices.tasks.legal_docs.reload_all_aos_wkly',
```

Comment:
The method name should say what it does `reload_all_aos`. The schedule should not leak into the method name. We would not want to change the method name if we decided to run this every morning at 5AM UTC.